theano/tensor/opt.py

```diff
@@ -2747,6 +2797,7 @@ def local_subtensor_merge(node):
 @register_specialize
 @gof.local_optimizer([Subtensor])
 def local_subtensor_of_alloc(node):
+    #TODO Julian: Document this better!
```

Comment:
I don't know how to document it. I will remove the comment for now, but you could add it as an issue if you want?